### PR TITLE
🎈Update select box colors and margins 🎈

### PR DESCRIPTION
**Pull request by wishdasher**
🔴 📙 💛 🍏 🔷

Updating colors for select box opener. Some margins got changed too.

https://zpl.io/V4JnRMp
zpl://screen?sid=5b060d842a9c0c332807a578&pid=5a622684dd6d876f24a8cb20

### CODE REVIEW

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit 67737cb99285973b7183a5bd05f98119873faeb7

https://deploy-preview-299--wonder-blocks.netlify.com

**Comment by kevinbarabash** [comment]
screen shot 2018-08-22 at 8 29 35 pm

The other openers on a white background have turned gray.  This screenshot is from Safari but something similar is happening in Firefox as well.

**Comment by wishdasher** [comment]
Ooh good catch, that's a default button style.

File: packages/wonder-blocks-dropdown/util/constants.js [hunk 1]
```diff
@@ -7,3 +9,8 @@ export const keyCodes = {
     up: 38,
     down: 40,
 };
+
+export const selectDropdownStyle = {
+    marginTop: Spacing.xSmall,
+    marginBottom: Spacing.xSmall,
+};
```
Comment (kevinbarabash) [inline packages/wonder-blocks-dropdown/util/constants.js]: nice

File: packages/wonder-blocks-dropdown/components/select-opener.js [hunk 2]
```diff
@@ -182,62 +172,83 @@ const styles = StyleSheet.create({
...
+                borderWidth: 2,
+                color: placeholder
+                    ? mix(fade(white, 0.32), blue)
+                    : mix(fade(blue, 0.32), white),
+                backgroundColor: mix(offBlack32, blue),
             },
             disabled: {
+                borderColor: mix(fade(white, 0.32), blue),
+                color: mix(fade(white, 0.32), blue),
                 cursor: "auto",
```
Comment (kevinbarabash) [inline packages/wonder-blocks-dropdown/components/select-opener.js]: Should we also be setting the `backgroundColor` for the disabled styles too?

File: packages/wonder-blocks-dropdown/components/select-opener.js [hunk 3]
```diff
@@ -145,34 +146,23 @@ const styles = StyleSheet.create({
...
-        color: offBlack32,
-    },
-
-    placeholder: {
-        color: offBlack50,
-    },
-
-    placeholderDisabled: {
-        color: offBlack16,
-    },
```
Comment (kevinbarabash) [inline packages/wonder-blocks-dropdown/components/select-opener.js]: Good call moving these down into the other styles.

**Pull request merged**